tensorboard/webapp/metrics/views/card_renderer/image_card_component.scss

```diff
@@ -112,6 +112,11 @@ $_second-row-height: 15px;
   position: relative;
 }
 
+.img-container img {
+  image-rendering: -moz-crisp-edges;
```

Comment:
attach a repro case or a logic in the comment so we do not revert this.

Reply:
Great idea, done.